tekton/pipeline_run.go

```diff
@@ -159,6 +160,13 @@ func (r *ReleasePipelineRun) WithReleaseStrategy(strategy *v1alpha1.ReleaseStrat
 	return r
 }
 
+// WithReleasePlanAdmission a reference to the ReleasePlanAdmission object to the pipeline
+func (r *ReleasePipelineRun) WithReleasePlanAdmission(planAdmission *v1alpha1.ReleasePlanAdmission) *ReleasePipelineRun {
+	r.WithObjectReferences(planAdmission)
```

Comment:
Nope. Just remove this whole method and use the existing call in `createReleasePipelineRun`.  Something like `WithObjectReferences(a.release, releasePlanAdmission)`.

Reply:
OK, I fixed it, it will be included in the second PR.